signing/src/main/java/tech/pegasys/web3signer/signing/config/SignerLoader.java

```diff
@@ -143,40 +202,57 @@ private MappedResults<ArtifactSigner> processMetadataFilesInParallel(
     }
   }
 
-  private MappedResults<ArtifactSigner> parseMetadataFiles(
-      final Map<Path, String> configFilesContents, final SignerParser signerParser) {
-    LOG.info("Parsing configuration metadata files");
+  private MappedResults<ArtifactSigner> mapToArtifactSignerSequentially(
+      final Collection<SigningMetadata> signingMetadataCollection,
+      final SignerParser signerParser) {
+    if (signingMetadataCollection.isEmpty()) {
+      return MappedResults.newSetInstance();
+    }
+
+    try {
+      return mapToArtifactSigner(signingMetadataCollection, signerParser, false);
+    } catch (final Exception e) {
+      return MappedResults.errorResult();
+    }
+  }
+
+  private MappedResults<ArtifactSigner> mapToArtifactSigner(
+      final Collection<SigningMetadata> signingMetadataCollection,
+      final SignerParser signerParser,
+      boolean useParallelStream) {
+    LOG.info("Converting signing metadata to Artifact Signer with parallel: {}", useParallelStream);
 
-    final Instant start = Instant.now();
     final AtomicLong configFilesHandled = new AtomicLong();
     final AtomicInteger errorCount = new AtomicInteger(0);
 
+    final Stream<SigningMetadata> entryStream =
```

Comment:
Could just pass in the appropriate stream then this statement wouldn't be needed

Reply:
good idea .... done.